windows/rke2-standalone-install.ps1

```diff
@@ -0,0 +1,103 @@
+<#
+.SYNOPSIS
+  Setup and Install standalone RKE2 in Windows to be used as Worker Nodes.
+.DESCRIPTION
+  Run the script to setup and install all RKE2 related needs.
+.Parameter ServerIP
+    Server IP of Primary server where rke2 is already installed and the worker will join.`
+.Parameter Token
+    Token of Primary server.`
+.Parameter Version
+    Version of rke2 to download from github.`
+.Parameter Commit
+    Commit of RKE2 to download from temporary cloud storage.
+    If set, this forces Method=tar.
+    * (for developer & QA use only)`
+.EXAMPLE
+  Usage:
+    Invoke-WebRequest ((New-Object System.Net.WebClient).DownloadString('https://github.com/rancher/rke2/blob/master/rke2-standalone-install.ps1'))
+    ./rke2-standalone-install.ps1 $ServerIP <server-IP> $Token <server-token> $Version <rke2-version>
+#>
+
+[CmdletBinding()]
+param (
+    [Parameter()]
+    [String]
+    $ServerIP,
+    [Parameter()]
+    [String]
+    $Version,
+    [Parameter()]
+    [String]
+    $Token,
+    [Parameter()]
+    [String]
+    $Commit
+)
+
+function Write-InfoLog() {
+    Write-Output "[INFO] $($args -join " ")"
+}
+
+function Write-WarnLog() {
+    Write-Output "[WARN] $($args -join " ")"
+}
+
+function Write-DebugLog() {
+    Write-Output "[DEBUG] $($args -join " ")"
+}
+
+# Set the version or commit based of cli
+function Set-Version() {
+    if ($Commit) {
+        $Version = "commit $($Commit)"
+    }
+    elseif ($Version) {
+        $Version = $Version
+    }
+    return $Version
+}
+
+function Enable-Features() {
+    Enable-WindowsOptionalFeature -Online -FeatureName Containers -All
+}
```

Comment:
Doesn't the node need to restart after this is enabled?

Reply:
I think there is a -Restart flag that will reboot it if necessary. If this is our YOLO script we might as well add that?

Reply:
As per the doc, it is required to reboot. But we saw rke2 was installed and started successfully even with no restart. So need to check if that is a hard requirement. I will check on this, thanks!